src/main/java/org/apache/sling/auth/oauth_client/impl/SlingUserInfoProcessorImpl.java

```diff
@@ -166,7 +166,7 @@ public SlingUserInfoProcessorImpl(
         // Store the Refresh Token on user node
         String refreshToken = tokens.accessToken();
         if (storeRefreshToken && refreshToken != null) {
-            credentials.setAttribute(OAuthTokenStore.PROPERTY_NAME_ACCESS_TOKEN, cryptoService.encrypt(refreshToken));
+            credentials.setAttribute(OAuthTokenStore.PROPERTY_NAME_REFRESH_TOKEN, cryptoService.encrypt(refreshToken));
```

Comment:
in cases where oak login tokens are being created credentials attributes will be stored in the token node. given that 'refreshToken' is encrypted, i don't see a huge risk. 
but this is something to be aware of.
maybe worth a documentation note....